velox/functions/sparksql/AtLeastNNonNulls.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/functions/Macros.h"
+
+namespace facebook::velox::functions::sparksql {
+
+template <typename T>
+struct AtLeastNNonNullsFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void callNullable(
+      out_type<bool>& out,
+      const arg_type<int32_t>* n,
```

Comment:
We can directly use const int32_t n to get the argument like this https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/DecimalFunctions.cpp#L246

Reply:
I think they don't support to mix the signatures of `call` and `callNullable`.
` error: static assertion failed: UDF must implement at least one of `call`, `callNullable`, or `callNullFree` functions.
`